public/app/isolation-controls.js

```
@@ -0,0 +1,344 @@
+  });
+}
+
+async function _recordVoiceprint(btn, statusEl) {
+  const stream  = IsolationControls.mediaStream;
+  const DURATION = 5000; // 5 seconds
+
+  _setVoiceprintStatus('Recording 5s…', 'recording');
+  btn.disabled = true;
+
+  try {
+    const recorder = new MediaRecorder(stream, { mimeType: 'audio/webm;codecs=opus' });
```

ESLint

[error] 'MediaRecorder' is not defined.

(no-undef)